spark/dl/src/main/scala/com/intel/analytics/bigdl/models/utils/ModelBroadcast.scala

```diff
@@ -57,7 +57,7 @@ class ModelBroadcast[T: ClassTag](implicit ev: TensorNumeric[T]) extends Seriali
    * @return model
    */
   def value(): Module[T] = {
-    val localModel = broadcastModel.value.cloneModule()
+    val localModel = broadcastModel.value.cloneModule(false).evaluate()
```

Comment:
We can eliminate `broadcastParameters`; just do
```
val localModel = broadcastModel.value.cloneModule(false)
```

BTW, why do we call `evaluate()` here?